trunk/SUAVE/Methods/Aerodynamics/Fidelity_Zero/Lift/compute_max_lift_coeff.py

```diff
@@ -119,7 +119,7 @@ def compute_max_lift_coeff(vehicle,conditions=None):
         Cd_ind += ( 0.01 ) * Swing / Sref
 
     Cl_max_ls = Cl_max_ls * max_lift_coefficient_factor
-    return Cl_max_ls, Cd_ind
+    return Cl_max_w_eng, Cl_max_ls, Cd_ind
```

Comment:
why not add the breakdown of contributions to the lift_breakdown? this would give you the info you need, and would also add more general information to the results without affecting other files.

...aerodynamics.lift_breakdown.clean = Cl_max_w_eng
...aerodynamics.lift_breakdown.flap = dcl_flap
...aerodynamics.lift_breakdown.slat = dcl_slat
...aerodynamics.lift_breakdown.fudge_factor = max_lift_coefficient_factor